virl2_client/models/lab.py

```diff
@@ -666,18 +694,18 @@ def _remove_link_local(self, link: Link) -> None:
     @check_stale
     @locked
     def remove_interface(
-        self, iface: Interface | str, wait: Optional[bool] = None
+        self, iface: Interface | str, wait: bool | None = None
     ) -> None:
         """
-        Removes an interface from the lab.
+        Remove an interface from the lab.
 
         If you have an interface object, you can also simply do::
 
             interface.remove()
 
-        :param iface: the interface object or ID
-        :param wait: Wait for convergence (if left at default,
-            the lab wait property takes precedence)
+        :param iface: The interface object or ID.
+        :param wait: A flag indicating whether to wait for convergence.
+            If left at the default value, the lab's wait property takes precedence.
```

Comment:
used instead. - 'takes precedence' to what?
Also, we _could_ in a different ticket default to _UNCHANGED, and support None as not wanting to wait no matter what lab has set. But that has compatibility implications.